qutebrowser/browser/urlmarks.py

```diff
@@ -186,7 +186,7 @@ def set_mark():
 
         if name in self.marks:
             message.confirm_async(
-                title="Override existing quickmark?",
+                title=f"Override existing quickmark?\nOld URL: {self.marks[name]}",
```

Comment:
The `\n` doesn't work in the title, and anyway it looks a bit better and is wrappable as body text
![longoverride1](https://github.com/qutebrowser/qutebrowser/assets/7052386/a7c85a6f-80ac-4aac-a0f7-07820256c094)
![longoverride2](https://github.com/qutebrowser/qutebrowser/assets/7052386/470cf5f5-b511-4557-81d0-bd9a807012d9)

```suggestion
                title=f"Override existing quickmark?",
                text=f"Old URL: {self.marks[name]}",

```